src/crate/theme/rtd/crate/layout.html

```diff
@@ -64,6 +70,16 @@
         </div>
         <div class="col-md-8 col-lg-9">
           <div class="wrapper-content-right">
+            {% if display_github %}
```

Comment:
Where is now `display_github` defined? Did you remove the definition?

Reply:
Yes, I wanted to test something and forgot to put it back. I just updated it.